podcasts/SuggestedFoldersView.swift

```diff
@@ -46,6 +46,9 @@ struct SuggestedFoldersView: View {
             Text(L10n.suggestedFoldersDescription)
                 .textStyle(SecondaryText())
             foldersView
+                .padding(.horizontal, -20)
```

Comment:
Is this padding related to the horizontal margin? If so, it might be worth defining it in a constant, especially if it's worth using the same number in the `SuggestedFolderPodcastView` - what do you think?